addons/account/models/chart_template.py

```diff
@@ -1080,6 +1095,44 @@ def get_repartition_line_create_vals(self, company):
             }))
         return rslt
 
+    def get_repartition_line_create_vals_complete(self, company):
+        """
+        This function returns a list of values to create the repartition lines of a tax based on
+        one or several account.tax.repartition.line.template. It mimicks the function get_repartition_line_create_vals
+        but adds some missing fields:
+        account_id: (account.account), invoice_tax_id: (account.tax), refund_tax_id: (account.tax)
+
+        Returns a list of (0,0, x) ORM commands to create the repartition lines starting with a (5,0,0)
+        command to clear the repartition.
+        """
+        rslt = self.get_repartition_line_create_vals(company)
+        for idx, template_line in zip(range(1, len(rslt)), self):  # ignore first ORM command
+            account_id = False
+            if template_line.account_id:
+                # take the first account.account which code begins with the correct code
+                try:
+                    account_id = self.env['account.account'].search([
+                        ('code', '=like', template_line.account_id.code + '%'),
+                        ('company_id', '=', company.id)
+                    ], limit=1).id
+                except IndexError:
```

Comment:
When is this raised ? To me, if we don't find anything, account_id will simply contain an empty recordset, not enter the "except" block.